packages/prisma/seed-app-store.ts

```diff
@@ -337,7 +337,9 @@ export default async function main() {
       invite_link: process.env.ZAPIER_INVITE_LINK,
     });
   }
-
+  await createApp("make", "make", ["automation"], "make_automation", {
+    invite_link: "https://make.com/en/hq/app-invitation/6cb2772b61966508dd8f414ba3b44510",
```

Comment:
i'll take a look at how we can get the app published, so we won't need the link no more

Reply:
There's a section to apply for review within the app I created on Make. I'll get it started and open another PR here so we can track the progress.
<img width="1426" alt="Screenshot 2023-09-01 at 9 15 15 AM" src="https://github.com/calcom/cal.com/assets/85004512/ba96df38-ff59-4a88-8031-2370bc183fb3">